util/app-config/src/tests/cli.rs

```diff
@@ -58,9 +58,6 @@ fn ba_message_requires_ba_arg_or_ba_code_hash() {
 
     let err = err.err().unwrap();
     assert_eq!(clap::error::ErrorKind::MissingRequiredArgument, err.kind());
-    assert!(err
-        .to_string()
-        .contains("The following required arguments were not provided"));
```

Comment:
I try to change the first word of `The` to `the`, then this testcase will success.